<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QnPictureSettingsDialog</class>
 <widget class="QDialog" name="QnPictureSettingsDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>599</width>
    <height>511</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_3">
   <item>
    <widget class="QStackedWidget" name="stackedWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="imagePage">
      <layout class="QVBoxLayout" name="verticalLayout">
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QnFramedLabel" name="imageLabel">
         <property name="lineWidth">
          <number>4</number>
         </property>
         <property name="text">
          <string>Image is not loaded</string>
         </property>
         <property name="scaledContents">
          <bool>true</bool>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="fisheyePage">
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QnFisheyeSettingsWidget" name="fisheyeWidget" native="true"/>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="fisheyeCheckBox">
     <property name="text">
      <string>This is a fish-eye lens image</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QnFisheyeSettingsWidget</class>
   <extends>QWidget</extends>
   <header>ui/widgets/properties/fisheye_settings_widget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>QnFramedLabel</class>
   <extends>QLabel</extends>
   <header location="global">ui/widgets/framed_label.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
